PagingWithNetworkSample/app/src/main/java/com/android/example/paging/pagingwithnetwork/reddit/ServiceLocator.kt

```diff
@@ -71,11 +71,11 @@ interface ServiceLocator {
 open class DefaultServiceLocator(val app: Application, val useInMemoryDb: Boolean) : ServiceLocator {
     // thread pool used for disk access
     @Suppress("PrivatePropertyName")
-    private val DISK_IO = Executors.newSingleThreadExecutor()
+    private val DISK_IO = Dispatchers.IO
```

Comment:
don't need the constants here since you're not declaring a new pool, can just return Dispatchers.IO

Reply:
Isn't this for testability? I would assume the right thing to do here would be to subclass the ServiceLocator and override the mocked / test version of things